contrib/runners/mistral_v2/tests/unit/test_mistral_v2.py

```diff
@@ -283,6 +283,81 @@ def test_launch_workflow(self):
     @mock.patch.object(
         executions.ExecutionManager, 'create',
         mock.MagicMock(return_value=executions.Execution(None, WF1_EXEC)))
+    def test_launch_workflow_under_parent_chain_with_jinja_params(self):
+        ac_ctx = {
+            'chain': {
+                'params': {
+                    'var1': 'foobar',
+                    'var2': '{{foobar}}',
```

Comment:
And while we are at it, let's also add some test cases for scenarios for variables which are composed of more complex strings and don't just contain a single Jinja expression and nothing else, e.g. `foo bar {{ var }} bar {{ varr }}{% if ...%}`, etc.